spec/models/dhs1171_pdf_spec.rb

```diff
@@ -39,7 +70,8 @@
     end
 
     it "returns the tempfile" do
-      snap_application = create(:snap_application)
+      member = create(:member)
+      snap_application = create(:snap_application, members: [member])
```

Comment:
Could you use `:with_member` here (and in other spots) where you are assigning it explicitly?